js/views/settingsVw.js

```diff
@@ -113,7 +114,19 @@ module.exports = Backbone.View.extend({
       self.delegateEvents(); //delegate again for re-render
       self.setFormValues();
       self.setState(self.options.state);
-      //self.renderBlocked(self.model.get("user").blocked);
+      
+      // Since the Blocked Users View kicks off many server calls (one
+      // for each blocked user) and since we are re-rendering the entire
+      // settings view often (after each save), we will cache the Blocked
+      // Users View. If for some reason, you really need to re-render it,
+      // call renderBlocked() explicitly after calling render().
```

Comment:
I was thinking hard-code the 25 and leave the rest as GUIDs, since that would take a few minutes and then defer the scroll based lazy load to sometime later.

Otherwise, the scroll based functionality would take me some hours to implement (maybe... 3 - 4ish). Is it enough of a priority, that I should go ahead with that?